src/python/espressomd/comfixed.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function, absolute_import
+from .script_interface import ScriptInterfaceHelper, script_interface_register
+
+@script_interface_register
+class ComFixed(ScriptInterfaceHelper):
+    """Fix the center of mass of specific types.
+
+    Subtracts mass-weighted fraction of the total
+    force action on all particles of the type from
+    the particles after each force calculation. This
+    keeps the center of mass of the type fixed iff
```

Comment:
The name of the function is comfix so why not remove all the momentum of the particles of the given type, even if the momentum for this type is nonzero initially?